packages/flutter/lib/src/services/image_resource.dart

```diff
@@ -16,13 +16,14 @@ typedef void ImageListener(ui.Image image);
 /// image object might change over time, either because the image is animating
 /// or because the underlying image resource was mutated.
 class ImageResource {
```

Comment:
should the listeners fire when the scale is changed?
should the scale not be allowed to change?

I don't really understand why the scale is mutable. once we support animated GIFs or whatever, surely they'll all have the same scale.